dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
```

Comment:
All this logic should go in the model that tracks assignments (`EnrollmentModuleAssignment`? `EnrollmentUnitAssignment`?).  Helpers should really only contain logic to help render view markup.